eclair-core/src/test/scala/fr/acinq/eclair/channel/FuzzySpec.scala

```diff
@@ -77,12 +77,14 @@ class FuzzySpec extends TestKitBaseClass with FixtureAnyFunSuiteLike with Channe
     within(30 seconds) {
       val aliceInit = Init(Alice.channelParams.initFeatures)
       val bobInit = Init(Bob.channelParams.initFeatures)
+      val aliceAlias = ShortChannelId.generateLocalAlias()
+      val bobAlias = ShortChannelId.generateLocalAlias()
```

Comment:
Would it make sense in these tests (and all the other tests affected by this PR) to use hard-coded local aliases instead? It will make debugging easier.